src/odbc.h

```diff
@@ -21,9 +21,7 @@
 
 #include <uv.h>
 #include <napi.h>
-#include <wchar.h>
-
-#include <algorithm>
+#include <wchar.h>s
```

Comment:
Dangling 's' here

Reply:
Strange there is a dangling 's' there. I don't see it in my code or in the master branch. But correct, if I don't include algorithm, it will not compile.